node/src/actors/rad_manager/handlers.rs

```diff
@@ -74,12 +74,14 @@ impl Handler<ResolveRA> for RadManager {
                     // Perform aggregation on the values that made it to the output vector after applying the
                     // source scripts (aka _normalization scripts_ in the original whitepaper) and filtering out
                     // failures.
-                    witnet_rad::run_aggregation_report(
+                    let (res, _ctx) = witnet_rad::run_aggregation_report(
```

Comment:
`_ctx` already means actor context, it may be confusing to give it the same name
```suggestion
                    let (res, _) = witnet_rad::run_aggregation_report(
```